### ISSUE TITLE
NLLLoss 

### ISSUE THREAD

**Issue by john2019-warwick** [AnubhavGupta3377/Text-Classification-Models#9]
Hi, thanks for your code! However, I think there might be a bug on NLLLoss: I think the input of nn.NLLloss() is after logsoftmax according to this manual: https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html, however, here only softmax is provided in model.py therefore, the loss is always negative. I only check the Transformer model.

Kind regards,
John
